www-src/app/replicator/init.coffee

```diff
@@ -0,0 +1,434 @@
+compareVersions = require('../lib/compare_versions').compareVersions
+
+log = require('../lib/persistent_log')
+    date: true
+    processusTag: "Init"
+
+module.exports = class Init
+
+    _.extend Init.prototype, Backbone.Events
+    _.extend Init.prototype, Backbone.StateMachine
+
+    # Override this function to use it as initialize.
+    startStateMachine: ->
+        @initialize()
+        Backbone.StateMachine.startStateMachine.bind(@)(arguments)
```

Comment:
@jacquarg It's the same as `Backbone.StateMachine.startStateMachine.apply @, arguments` ?